src/javascript/_common/base/login.js

```diff
@@ -8,11 +8,15 @@ const urlForCurrentDomain = require('../url').urlForCurrentDomain;
 const isLoginPages        = require('../utility').isLoginPages;
 const TrafficSource       = require('../../app/common/traffic_source');
 const getAppId            = require('../../config').getAppId;
+const Url                 = require('../../../javascript/_common/url');
```

Comment:
why are you importing it in such a weird way?

```suggestion
const Url                 = require('../url');
```
this should work, same as line 7, and you can replace line 7 with it and use `Url.urlForCurrentDomain` throughout the file instead

Reply:
Done